08-mapping.Rmd

```diff
@@ -53,24 +53,22 @@ Finally, Section \@ref(other-mapping-packages) covers a range of alternative map
+Things have advanced greatly since the release of **sp** [see @pebesma_classes_2005] and many new techniques for map making have been developed.
+However, a decade later static plotting was still the emphasis of geographic data visualisation in R [@cheshire_spatial_2015] and still is its foundation despite the innovation of interactive mapping.
+
+The generic `plot()` function is often the fastest way to create static maps from vector and raster spatial objects (see sections \@ref(basic-map) and \@ref(basic-map-raster)).
+Sometimes, simplicity and speed are priorities, especially during the development phase of a project, and this is where `plot()` excels.
 The base R approach is also extensible, with `plot()` offering dozens of arguments.
-Another low-level approach is the **grid** package, which provides functions for low-level control of graphical outputs, --- see *R Graphics* [@murrell_r_2016], especially  Chapter [14](https://www.stat.auckland.ac.nz/~paul/RG2e/chapter14.html).
-The focus of this section, however, is making static maps with **tmap**, emphasizing the important aesthetic and layout options.
-
-Why **tmap**?
-It is a powerful and flexible map-making package with sensible defaults.
-It has a concise syntax that allows for the creation of attractive maps with minimal code, which will be familiar to **ggplot2** users.
-Furthermore, **tmap** has a unique capability to generate static and interactive maps using the same code via `tmap_mode()`.
-It accepts a wider range of spatial classes (including `raster` objects) than alternatives such as **ggplot2**, as documented in vignettes [`tmap-getstarted`](https://cran.r-project.org/web/packages/tmap/vignettes/tmap-getstarted.html) and [`tmap-changes-v2`](https://cran.r-project.org/web/packages/tmap/vignettes/tmap-changes-v2.html) and an academic paper on the subject [@tennekes_tmap_2018].
+Another low-level approach is the **grid** package which provides functions for basic control of graphical outputs---see *R Graphics* [@murrell_r_2016], especially  Chapter [14](https://www.stat.auckland.ac.nz/~paul/RG2e/chapter14.html). 
+This section however focuses on **tmap** and emphasizes the important aesthetic and layout options.
+
+**tmap** is a powerful and flexible map-making package with sensible defaults.
+It has a concise syntax that allows for the creation of attractive maps with minimal code which will be familiar to **ggplot2** users.
+It also has the unique capability to generate static and interactive maps using the same code via `tmap_mode()`.
+Finally, it accepts a wider range of spatial classes (including `raster` objects) than alternatives such as **ggplot2**.
+The vignettes [`tmap-getstarted`](https://cran.r-project.org/web/packages/tmap/vignettes/tmap-getstarted.html) and [`tmap-changes-v2`](https://cran.r-project.org/web/packages/tmap/vignettes/tmap-changes-v2.html), as well as Tennekes ([@tennekes_tmap_2018])' academic paper on the subject document some of these capabilities.
```

Comment:
Not sure the double 'Tennekes' is needed here.

Reply:
Oh, my bad: I thought that this would get rendered into `Tennekes (2018)`. Since the hyperlink is only on `2018` in the current version, I thought that `[@tennekes_tmap_2018]` would get rendered into `2018` only. I don't use markdown :tongue:

Reply:
Will definitely correct this tomorrow.

Reply:
What would the proper way to render `Tennekes (2018)` be then? (rather than `Tennekes 2018`). I was hoping to write `Tennekes (2018)' academic paper...`

Reply:
Like this:

```
as well as @tennekes_tmap_2018.
```

I don't think it needs to state that it's an academic paper. Kind of irrelevant in this context. So suggest changing to something like this:

```
The see the vignettes [`tmap-getstarted`](https://cran.r-project.org/web/packages/tmap/vignettes/tmap-getstarted.html) and [`tmap-changes-v2`](https://cran.r-project.org/web/packages/tmap/vignettes/tmap-changes-v2.html), as well as @tennekes_tmap_2018, for further documentation.
```

Reply:
I agree. I had kept it to stay closer to the original text, but i like your suggestion here much better.

Reply:
> as well as @tennekes_tmap_2018

Would this not lead to `Tennekes 2018`? (so without the parenthesis around `2018`).

I am wondering about that since `[see @pebesma_classes_2005]` gets rendered into `(see Pebesma and Bivand 2005)`. It does not matter any longer here, but I was just curious how to get the `author (year)` format with markdown.

Reply:
Oh, I guess the `[ ]` suppress the parentheses around the year, but without `[ ]` it would be there. That would explain the various cases.